app/controllers/contacts.js

```diff
@@ -0,0 +1,102 @@
+import BaseController from "./base/base.js";
+import Contact from "./../models/contact.js";
+import ContactsMatchRepository from "../repositories/contact_match_repository.js";
+import SessionRepository from "./../repositories/session_repository.js";
+import { ACTIVE } from "./../store/session.js";
+import { ERROR_STATUES } from "./../validations/constants/errors.js";
+import { ObjectId } from "mongodb";
+
+class ContactsController extends BaseController {
+  constructor() {
+    super();
+
+    this.sessionRepository = new SessionRepository(ACTIVE);
+    this.contactMatchRepository = new ContactsMatchRepository();
+  }
+
+  async contact_add(ws, data) {
+    const { id: requestId, contact_add: contactData } = data;
+    const currentUser = this.sessionRepository.getSessionUserId(ws);
+
+    await this.contactMatchRepository.matchedContactWithUser(contactData);
```

Comment:
can we please use `match` instead of `matched`. The rest looks good in this file.